managedtenants/utils/ocm.py

```diff
@@ -277,6 +316,15 @@ def _addon_from_imageset(self, imageset, metadata):
                 addon[self.IMAGESET_KEYS[key]] = val
         return addon
 
+    @staticmethod
+    def _make_additional_ctlg_src_list(ctlg_srcs):
```

Comment:
Could `_make_additional_ctlg_src_list` and`_make_env_vars_list` be combined into a single function?

Reply:
It would make a nice one liner too:
```python
def index_dicts(dicts):
    return [dict(d, id=str(id)) for id, d in enumerate(dicts)]
```